### ISSUE TITLE
normal subscribers can't upload a profil image

### ISSUE THREAD

**Issue by astroidex** [OSGeo/osgeo#184]
* normal user can't upload an Image (report was: The "add image" button does nothing")
 *  I tried it with the account of my collegue theklaw (Subscriber) and noticed that she can't upload an image

Problem
* click on button -> nothing happens -> no error message

see also #183#issuecomment-359673050 and #178

**Comment by robe2** [comment]
I can change the permission of subscribers to upload images, but there doesn't seem to be any granularity I can do beyond that which I guess means they can upload anywhere they can edit.

If folks are okay with me making this change to subscriber role, I can do it.

**Comment by astroidex** [comment]
Thanks @robe2 for the feedback. Not so nice. Maybe we should have another at the #183 gravatar solution. On the other hand in our wiki a user can edit every page too.

What do you think? @cvvergara @jodygarnett @wildintellect ?

**Comment by jodygarnett** [comment]
I am really struggling with the different levels, subscriber cannot do much, ... but editor can delete other peoples pages.

I think we need an orientation session on wordpress security roles.

**Comment by robe2** [comment]
@jodygarnett  yes agreed.  I have little idea how wordpress security roles work.  Doesn't seem to be much granularity (e.g. what section one can edit) in the role admin options.

**Comment by robe2** [comment]
Looking at roles closer, I think Author might be the right level of granularity we need.

Author is a lower level than Editor.  And has 7 permissions:

1) General: Can read (I presume anything not marked as private)
2) Media: Can Upload files
3)  Pages:  Can Read  (I note there is a Edit Pages, and Edit Other's pages that is not checked.  So I assume maybe Edit Pages only allows editing of your own pages (same options for delete)
4)  Posts: Can (Read | Edit | Delete (and published) ) Posts ( (Edit | Delete) other's posts is unchecked)

**Comment by jodygarnett** [comment]
I am concerned that the gravatar is doing more harm then good now, people see their face and do not notice they need to upload an image. We have a lot of "empty" faces showing up when looking at projects or the members list.

**Comment by robe2** [comment]
I think rather than using a built-in might be better to just create our own role, maybe call it 

OSGeo Basic

I'm hesitant to muck with existing roles since the level of permission isn't entirely clear from the names.

So was thinking OSGeo Basic for starters would just be able to do what subscriber does plus upload images.  We might want to add more permissions after.

**Comment by cvvergara** [comment]
@robe2 
does not matter what role name they use, the issues I mentioned on the report about subscriber apply to all roles.

right now we can give permission to the subscriber to upload an image and "disable" avatar, so when we have made progress on using the avatar, we "enable" it again?

**Comment by cvvergara** [comment]
@astroidex 
As far as I can remember this issue is solved, can you please verify.
